src/nvim/screen.c

```diff
@@ -3469,7 +3469,7 @@ win_line (
                            && !(noinvcur
                                 && lnum == wp->w_cursor.lnum
                                 && (colnr_T)vcol == wp->w_virtcol)))
-                   && lcs_eol_one >= 0) {
+                   && lcs_eol_one > 0) {
           /* Display a '$' after the line or highlight an extra
```

Comment:
Lint is failing because of the comment style. These need to be changed to `//`.

You can run `make lint` locally to check if you've fixed all the linter errors.